### DISCUSSION TITLE
How I prevent the Telescope Previewer from inheriting some vimrc settings?

### DISCUSSION THREAD

**Discussion by mystilleef** [nvim-telescope/telescope.nvim#1377] [Q&A]
How do I prevent the previewer from using my `vimrc` `number`, `relativenumber`, and `numberwidth` settings?

I've tried the following.

`au User TelescopePreviewerLoaded setlocal numberwidth=1`

It doesn't work. 

Does the Telescope Previewer have a `filetype` like the Telescope Prompt that I can manipulate with `autocmds`?

### Screenshot

![Screenshot from 2021-10-23 20-50-38](https://user-images.githubusercontent.com/273399/138611071-4d875bcd-cf6e-45b5-a883-6656abebde59.png)

### Version

```sh
NVIM v0.6.0-dev+507-g9f392c071
Build type: Release
LuaJIT 2.1.0-beta3
```

**Comment by fdschmidt93** [comment]
Sorry for the late reply. This should only affect the buffers picker I presume? The buffers picker right now shows the actually opened buffers as they are such that special buffers like terminals can also be previewed.

However, this causes a lot of issues. We plan on reverting to the same kind of previewer as for find_files or live_grep.

E: sorry, I've overlooked who posted this :) you are already aware :sweat_smile:

**Comment by mystilleef** [comment]
#1400 fixed this problem for me. Once again, thanks.
